jemalloc-sys/build.rs

```diff
@@ -155,6 +155,17 @@ fn main() {
             .expect("failed to copy config file to OUT_DIR");
     }
 
+    // Create a dummy .git file so that 'git describe' fails and
+    // configure uses the VERSION file we provide.
+    if let Err(e) = fs::File::create(build_dir.join(".git")) {
```

Comment:
You can pass the version to build script by `--with-version=xxx` instead.

Let's skip and delete the VERSION file in configure directory and only use version from CARGO_PKG_VERSION. So it's easier to maintain.

Reply:
done. I also updated the metadata part of the version so that still get the same result.